### PR TITLE
docs: add Node Docker deploy example

### PR DESCRIPTION
**Pull request by nelsonprsousa**
# Overview

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [X] Docs / tests / types / typos

# Description

Added a doc section on how to run a Qwik app using Docker.

# Checklist:

- [X] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by gioboa** [comment]
Thanks @nelsonprsousa for your help. I love the idea to share a Dockerfile example.
My only concerns is that this is more a cookbook example. Can you move your code [there](https://qwik.builder.io/docs/cookbook/)?
We can mention this example in the node deployments page though

**Review by gioboa — CHANGES_REQUESTED** [review]
Thanks @nelsonprsousa for your help. I love the idea to share a Dockerfile example.
My only concerns is that this is more a cookbook example. Can you move your code [there](https://qwik.builder.io/docs/cookbook/)?
We can mention this example in the node deployments page though

**Review by mhevery — APPROVED** [review]

**Comment by mhevery** [comment]
+1 for moving to cookbook.

**Comment by gioboa** [comment]
@nelsonprsousa do you have any news for this one?

**Comment by nelsonprsousa** [comment]
> @nelsonprsousa do you have any news for this one?

Still on my _todo_ list. Once the festives are over count on me to close this one 👍

**Comment by nelsonprsousa** [comment]
> @nelsonprsousa do you have any news for this one?

Feel free to give some feedback.

cc @mhevery @gioboa

File: packages/docs/src/routes/docs/cookbook/index.mdx [hunk 1]
```diff
@@ -22,4 +22,4 @@ Examples:
 - [Re-exporting loaders](./re-exporting-loaders/)
 - [Theme Managment](./theme-management/)
 - [NavLink](./nav-link/)
-
+- [Deploy with Node using Docker](./node-docker-deploy)
```
[the remark this answers is not in the document]
Comment (gioboa) [reply packages/docs/src/routes/docs/cookbook/index.mdx:25]: You need to add this page in this file too
`qwik/packages/docs/src/routes/docs/menu.md`
Thanks

**Review by gioboa — CHANGES_REQUESTED** [review]

File: packages/docs/src/routes/docs/cookbook/index.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (nelsonprsousa) [reply packages/docs/src/routes/docs/cookbook/index.mdx]: Done and done, can you please check? 🙏

File: packages/docs/src/routes/docs/cookbook/node-docker-deploy/index.mdx [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
+title: Cookbook | Deploy with Node using Docker
+contributors:
+  - nelsonprsousa
+---
```
[the remark this answers is not in the document]
Comment (nelsonprsousa) [reply packages/docs/src/routes/docs/cookbook/node-docker-deploy/index.mdx:7]: Most of the other cookbook docs do not have that. However I added on the new one.

Can you please check? 🙏

File: packages/docs/src/routes/docs/cookbook/node-docker-deploy/index.mdx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+ ```shell
+ docker build -t your-image .
+ ```
+
+And start a Docker container:
+
+ ```shell
+docker run -dp 127.0.0.1:3000:3000 your-image
+ ```
```
[the remark this answers is not in the document]
Comment (nelsonprsousa) [reply packages/docs/src/routes/docs/cookbook/node-docker-deploy/index.mdx:92]: Did my best. Would like to receive some feedback.

[the remark this answers is not in the document]
Comment (gioboa) [reply packages/docs/src/routes/docs/cookbook/node-docker-deploy/index.mdx:92]: That's awesome, your rock 🚀

**Review by gioboa — CHANGES_REQUESTED** [review]

File: packages/docs/src/routes/docs/cookbook/node-docker-deploy/index.mdx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+title: Cookbook | Deploy with Node using Docker
+contributors:
+  - nelsonprsousa
+updated_at: '2023-12-28T16:00:00Z'
+created_at: '2023-12-28T16:00:00Z'
+---
+
+# Deploy with Node using Docker
+
+This Dockerfile is used to build a Docker image for our Qwik Node.js application. You can edit it accordingly to use npm, pnpm or bun.
```
[the remark this answers is not in the document]
Comment (gioboa) [reply packages/docs/src/routes/docs/cookbook/node-docker-deploy/index.mdx:11]: With this little change I think will be perfect 😊

[the remark this answers is not in the document]
Comment (nelsonprsousa) [reply packages/docs/src/routes/docs/cookbook/node-docker-deploy/index.mdx]: Just added a phrase saying that, as sugeested. We can in the future add more examples for other package managers if this isn't enough 👍

**Comment by gioboa** [comment]
The menu label was too long, I fixed it.
Screenshot 2023-12-28 at 21 46 44

**Review by gioboa — APPROVED** [review]
Thanks @nelsonprsousa

**Comment by nelsonprsousa** [comment]
> The menu label was too long, I fixed it. Screenshot 2023-12-28 at 21 46 44
> 
> Screenshot 2023-12-28 at 21 53 28

Thanks. I am not aware on how to check a preview of the website?

**Comment by gioboa** [comment]
From the root  install the dependencies and run
```
pnpm docs.dev
```
👍

**Comment by gioboa** [comment]
Yes, it is. You need to change yarn with bun and use the bun adapter to build your app

**Pull request merged**